runbotics/runbotics-scheduler/src/mail/mail.service.ts

```diff
@@ -53,18 +57,32 @@ export class MailService {
     }
 
     public async sendBotDisconnectionNotificationMail(bot: IBot, installationId: string) {
-        await this.sendMail({
-            to: bot.user.email ?? '',
-            subject: NOTIFICATION_MAIL_SUBJECT,
-            content: `Hello,\n\nBot 🤖 (${installationId}) has been disconnected.\n\nBest regards,\nRunBotics`,
-        });
+        const disconnectedBot = await this.botService.findById(bot.id);
+
+        if (disconnectedBot.subscribers && disconnectedBot.subscribers.length) {
+            const subscribers = disconnectedBot.subscribers;
+            const emailAddresses = subscribers.map(({ email }) => email).join(',');
+
+            await this.sendMail({
+                to: emailAddresses ?? '',
```

Comment:
I know it was here before but it does not make sense to send an email to ''.
imo this shoud be mandatory or email should be sent to the creator of the process